### ISSUE TITLE
[Functional] [Keyboard- Add QnA maker] The "Connect to QnA...." button is not functioning as well as after clicking on the same, "+" button in the service drop down is also not accessible with mouse nor with keyboard.

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1906]
**Actual Result:**
The "Connect your bot to a QnA Maker Knowledge base manually" button is not functioning as well as after clicking on the same, "+" button in the service drop down is also not accessible with mouse nor with keyboard.

**Expected Result:**
As it is defined as button with some specific function, it's functionality should be accessible by keyboard or mouse. As well as it shouldn't block the functionality of other controls i.e. "+" button in "Service" drop down.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to the "Service" and click on "+".
4. Select "Add QnA Maker" from appeared "Menu".
5. Navigate to "Connect your bot to a QnA Maker Knowledge base manually" button and try to activate it with keyboard or mouse.
6. Try to perform step-3, again.
7. Observe the issue.

**Note:**
Same issue also observed in Linux system.

These links/button are also not functioning:
1. "Learn more about keys in LUIS" button. {Repro Step: Bot Explorer->Services->+symbol->Add language understanding-> add a LUIS app manually}
2. "Jump to Bot state Inspector" and "Update your Bot's code" links. {Repro Steps: Help->Get started with channels}

**User Impact:**
The keyboard dependent user will get confused if the links/buttons didn't serve their purpose. They are not be able to explore the content defined in the link/button.

**Test Environment:**
OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (Version 4.5.2-80829)

OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.5.2-linux

**Attachment:**
[Button or Links is not functioning.zip](https://github.com/microsoft/BotFramework-Emulator/files/3690345/Button.or.Links.is.not.functioning.zip)
[Button or Links is not functioning_Linux.zip](https://github.com/microsoft/BotFramework-Emulator/files/3819237/Button.or.Links.is.not.functioning_Linux.zip)

**Comment by ShikhaMishra11** [comment]
Verified the issue and it is fixed in latest build.
OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (version: 4.6.0-92058)

OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.6.0-92058-linux-x86_64
